gst/tensor_converter/tensor_converter.c

```diff
@@ -641,6 +641,8 @@ gst_tensor_converter_chain (GstPad * pad, GstObject * parent, GstBuffer * buf)
 
         /** copy timestamps */
         gst_buffer_copy_into (inbuf, buf, GST_BUFFER_COPY_METADATA, 0, -1);
+
+        gst_buffer_unref (buf);
```

Comment:
Is this really ok? Where did we increase ref-count of buf in this function?

Reply:
In tensor-converter, we don't increase the ref count of input buf.
In this case, converter creates new buffer and copies data in buf, and then pushes newly created buffer into source pad.

Reply:
Does COPY_METADATA include Timestamp? If you want to copy timestamps, then prefer GST_BUFFER_COPY_TIMESTAMPS?

Reply:
Ok. Got it: https://gstreamer.freedesktop.org/data/doc/gstreamer/head/gstreamer/html/GstPad.html#GstPadChainFunction

Reply:
Then what about buf of ```_NNS_AUDIO``` or ```octet```? Don't you need to unref it as well?